lms/static/sass/_build-course.scss

```diff
@@ -30,7 +30,6 @@
 @import 'course/modules/student-notes'; // student notes
 @import 'course/modules/calculator'; // calculator utility
 @import 'course/modules/timer'; // timer
-@import 'course/modules/chat'; // chat utility
```

Comment:
I think the implication here is that this was including the contents of the feature's stylesheet even when the feature was disabled. The file isn't huge, but still, wasted bytes across the wire..

eek! or am I missing something?